src/components/search/concept-modal/concept-modal.js

```diff
@@ -214,13 +217,14 @@ export const ConceptModalBody = ({ result }) => {
           return studies
         }, {}))
       } catch (e) {
-        if (e.name !== "CanceledError") throw e
+        if (e.name !== "CanceledError") {
+          console.warning(e)
+          setStudies(null)
```

Comment:
Would it be useful to propagate the exception?

Reply:
Not in this scenario. The exception is "implicitly" handled by setting the state to null, indicating an error state.